app/Models/Permission.php

```diff
@@ -44,7 +44,7 @@ public function setNameAttribute($value)
 
     public function getSubPermissionAttribute()
     {
-        return ($this->attributes['fid'] == 0) ? $this->where('fid',$this->attributes['id'])->orderBy('sort', 'asc')->get() : null;
+        return (empty($this->attributes['parent'])) ? $this->where('parent', basename($this->attributes['action']))->orderBy('sort', 'asc')->get() : null;
```

Comment:
public function chlidren(){
     return $this->hasMany('permissions', 'parent', 'action');
}